.nvmrc

```diff
@@ -1 +1 @@
-12.5.0
\ No newline at end of file
+10.*
```

Comment:
Is this offical now? Cause I'll copy it over to the two lambdas that I've done if it is.

Reply:
it's been agreed that it will be looked when upgrading lambda runtime and to keep it as is
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html